WireDomain/Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
         .target(
             name: "WireDomainPkg",
             path: "./Sources/WireDomain",
-            sources: ["./UseCases/Protocols/IndividualToTeamMigrationUseCaseProtocol.swift"]
+            sources: ["./UseCases/Protocols"]
```

Comment:
We still have a `WireDomain` Xcode project due to code depending on `WireDataModel` etc, but also have a `WireDomainPkg`, for domain specific code which can be used in other Swift packages.
Both targets reside in the same directories on the file system.

This workaround explicitly includes the protocols in the SPM target, while they are explicitly excluded in the Xcode project.

Reply:
Exactly.